core/include/traccc/fitting/kalman_filter/two_filters_smoother.hpp

```diff
@@ -175,7 +175,7 @@ struct two_filters_smoother {
 
         // Calculate backward chi2
         const matrix_type<D, D> R = (I_m - H * K) * V;
-        assert(matrix::determinant(R) != 0.f);
+        // assert(matrix::determinant(R) != 0.f);
```

Comment:
is this necessary?

Reply:
Due to PR https://github.com/acts-project/detray/pull/977 in detray, the test track that are used will be slightly different tracks than before